mods/ts/rules/defaults.yaml

```diff
@@ -417,6 +417,8 @@
 		Weapons: SmallDebris
 		Pieces: 3, 7
 		Range: 2c0, 5c0
+	PoisonedByTiberium:
+		Resources: Veins
```

Comment:
I have no idea what `VeinDamage=5` has to be translated to. The default damage makes sense. It damages vehicles slowly.